src/drt/src/dr/FlexGridGraph.h

```diff
@@ -939,18 +957,12 @@ class FlexGridGraph
     // Byte6
     frUInt4 fixedShapeCostVia : cost_bits;
     // Byte7
-    frUInt4 fixedShapeCostPlanar : cost_bits;
-
-    template <class Archive>
-    void serialize(Archive& ar, const unsigned int version)
-    {
-      uint64_t* val = reinterpret_cast<uint64_t*>(this);
-      (ar) & *val;
-    }
-    friend class boost::serialization::access;
+    frUInt4 fixedShapeCostPlanarHorz : cost_bits;
+    // Byte8
+    frUInt4 fixedShapeCostPlanarVert : cost_bits;
   };
 #ifndef DEBUG_DRT_UNDERFLOW
-  static_assert(sizeof(Node) == 8);
+  static_assert(sizeof(Node) == 12);
```

Comment:
how much does this impact overall memory usage?

Reply:
This could cause an adverse impact - 8 bytes is very nice for cache locality. 9 bytes might cause a lot of extra padding.

Reply:
No impact on memory. ISPD19_TEST9:
```
master:
[INFO DRT-0267] cpu time = 40:52:30, elapsed time = 02:23:33, memory = 27653.59 (MB), peak = 31375.72 (MB)
this branch:
[INFO DRT-0267] cpu time = 36:49:32, elapsed time = 02:08:23, memory = 27678.93 (MB), peak = 31399.07 (MB)

```

Reply:
That's quite surprising but good news